spacy/lang/grc/__init__.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+from thinc.api import Model
+from .tokenizer_exceptions import TOKENIZER_EXCEPTIONS
+from .stop_words import STOP_WORDS
+from .lex_attrs import LEX_ATTRS
+from ...language import Language
+
+
+
```

Comment:
Could you format your files with [`black`](https://black.readthedocs.io/en/stable/)? It will automatically make sure the format adheres to the rest of the code base - e.g. using only 2 newlines here instead of 3.